dpsim/include/dpsim/MNASolverDirect.h

```diff
@@ -74,6 +75,8 @@ namespace DPsim {
 		std::shared_ptr<DirectLinearSolver> mDirectLinearSolverVariableSystemMatrix;
 		/// LU factorization indicator
 		DirectLinearSolverImpl implementationInUse;
+		/// LU factorization configuration
+		DirectLinearSolverConfiguration configurationInUse;
```

Comment:
Use for member vars `mImplementationInUse` and `mConfigurationInUse`

Reply:
done